src/App.jsx

```diff
@@ -14,57 +15,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [message, setMessage] = useState('Jam is selected');
```

Comment:
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```
just save the good name